script.js

```diff
@@ -0,0 +1,137 @@
+var newGameBtn = document.getElementById('js-newGameButton');
+
+newGameBtn.addEventListener('click', newGame);
+
+var pickRock = document.getElementById('js-playerPick_rock'),
+     pickPaper = document.getElementById('js-playerPick_paper'),
+     pickScissors = document.getElementById('js-playerPick_scissors');
+
+pickRock.addEventListener('click', function() { playerPick('Rock') });
+pickPaper.addEventListener('click', function() { playerPick('Paper') });
+pickScissors.addEventListener('click', function() { playerPick('Scissors') });
+
+var gameState = 'notstarted',  //started // ended
+    player = {
+        name: '',
+        score: 0
+    },
+    computer = {
+        score: 0
+    };
+
+ var newGameElem = document.getElementById('js-newGameElement'),
+    pickElem = document.getElementById('js-playerPickElement'),
+    resultsElem = document.getElementById('js-resultsTableElement');
+
+
+function setGameElements() {
+  switch(gameState) {
+    case 'started':
+        newGameElem.style.display = 'none';
+        pickElem.style.display = 'block';
+        resultsElem.style.display = 'block';
+      break;
+    case 'ended':
+        newGameBtn.innerText = 'Jeszcze raz';
+    case 'notStarted':
+    default:
+        newGameElem.style.display = 'block';
+        pickElem.style.display = 'none';
+        resultsElem.style.display = 'none';
+  }
+}
+
+setGameElements();
+
+var playerPointsElem = document.getElementById('js-playerPoints'),
+    playerNameElem = document.getElementById('js-playerName'),
+    computerPointsElem = document.getElementById('js-computerPoints');
+
+function newGame() {
+  player.name = prompt('Please enter your name');
+  if (player.name) {
```

Comment:
try to do the same without writing everything inside your if statement :) tip: use negation operator and return